client/src/templates/Challenges/redux/execute-challenge-saga.js

```diff
@@ -167,7 +167,11 @@ function* executeTests(testRunner, tests, testTimeout = 5000) {
         throw err;
       }
     } catch (err) {
-      newTest.message = text;
+      const { actual, expected } = err;
+
+      newTest.message = text
+        .replace('--fcc-expected--', expected)
+        .replace('--fcc-actual--', actual);
```

Comment:
The above just looks like this:
```suggestion
      newTest.message = `${text}\n\n${message}`;
```

Reply:
Either that, or simply the following which can easily be translated:
```suggestion
      const translatableAssertion = `Expected ${expected}, but found ${actual}`;
      newTest.message = `${text}\n\n{translatableAssertion}`;
```

Reply:
Thing is, what happen if you don't want to say exactly "Expected ..., but found ..."?  

Also, that locks into either outputting this for every test or adding more code (frontmatter, maybe?) to configure which tests show it.

For what it's worth
```
Your `h1` element should have the text `Hello World`.
Expected --fcc-expected--, but found --fcc-actual--. 
```

works with the current parser.

Reply:
> Thing is, what happen if you don't want to say exactly "Expected ..., but found ..."?

This is why I would prefer sticking with relying on the actual assertion message from specific Chai methods.

> more code (frontmatter, maybe?)

Maybe? Either `--fcc-actual--...` or some flag at the end of a hint `--fcc-show-assertion--`

Reply:
`--fcc-show-assertion--`, yeah, that's where my mind went.  I think that's a reasonable approach.